doc/manuals/orion-api.md

```diff
@@ -1,127 +1,81 @@
+    - [Attribute names restrictions](#attribute-names-restrictions)
+    - [Metadata names restrictions](#metadata-names-restrictions)
+    - [Ordering Results](#ordering-results)
+    - [Error Responses](#error-responses)
+    - [Geospatial properties of entities](#geospatial-properties-of-entities)
+        - [Simple Location Format](#simple-location-format)
+        - [GeoJSON](#geojson)
+    - [Simple Query Language](#simple-query-language)
+    - [Geographical Queries](#geographical-queries)
+        - [Query Resolution](#query-resolution)
+    - [Filtering out attributes and metadata](#filtering-out-attributes-and-metadata)
+    - [Notification Messages](#notification-messages)
+    - [Custom Notifications](#custom-notifications)
+    - [Group API Entry Point](#group-api-entry-point)
+        - [Retrieve API Resources [GET /v2]](#retrieve-api-resources-get-v2)
+- [API Routes](#api-routes)
+    - [Entities Operations](#entities-operations)
+        - [Entities List](#entities-list)
+            - [List Entities [GET /v2/entities]](#list-entities-get-v2entities)
+            - [Create Entity [POST /v2/entities]](#create-entity-post-v2entities)
+        - [Entity by ID](#entity-by-id)
+            - [Retrieve Entity [GET /v2/entities/]](#retrieve-entity-get-v2entities)
+            - [Retrieve Entity Attributes [GET /v2/entities/{entityId}/attrs]](#retrieve-entity-attributes-get-v2entitiesentityidattrs)
+            - [Update or Append Entity Attributes [POST /v2/entities/{entityId}/attrs]](#update-or-append-entity-attributes-post-v2entitiesentityidattrs)
+            - [Update Existing Entity Attributes [PATCH /v2/entities/{entityId}/attrs]](#update-existing-entity-attributes-patch-v2entitiesentityidattrs)
+            - [Replace all entity attributes [PUT /v2/entities/{entityId}/attrs]](#replace-all-entity-attributes-put-v2entitiesentityidattrs)
+            - [Remove Entity [DELETE /v2/entities/{entityId}]](#remove-entity-delete-v2entitiesentityid)
+        - [Attributes](#attributes)
+            - [Get attribute data [GET /v2/entities/{entityId}/attrs/{attrName}]](#get-attribute-data-get-v2entitiesentityidattrsattrname)
+            - [Update Attribute Data [PUT /v2/entities/{entityId}/attrs/{attrName}]](#update-attribute-data-put-v2entitiesentityidattrsattrname)
+            - [Remove a Single Attribute [DELETE /v2/entities/{entityId}/attrs/{attrName}]](#remove-a-single-attribute-delete-v2entitiesentityidattrsattrname)
+        - [Attribute Value](#attribute-value)
+            - [Get Attribute Value [GET /v2/entities/{entityId}/attrs/{attrName}/value]](#get-attribute-value-get-v2entitiesentityidattrsattrnamevalue)
+            - [Update Attribute Value [PUT /v2/entities/{entityId}/attrs/{attrName}/value]](#update-attribute-value-put-v2entitiesentityidattrsattrnamevalue)
+        - [Types](#types)
+            - [List Entity Types [GET /v2/types/]](#list-entity-types-get-v2types)
```

Comment:
It some places we have, eg.

* `[PUT /v2/entities/{entityId}/attrs/{attrName}/value`, i.e. without trailing slash
* `[GET /v2/types/]`, i.e. with trailing slash

Let's make all the same way and, from my point of view, is better the one "without trailing slash" (easier to read, less symbols :)